docs/components/ColorSwatchGroup/ColorSwatchGroup.tsx

```diff
@@ -1,5 +1,6 @@
-import React, { FunctionComponent } from 'react';
```

Comment:
Includes all the changes from https://github.com/MetaMask/design-tokens/pull/698. I think it would be good to get these into main so we can separate documentation UI updates from actual token color changes.